pkg/artifact.go

```diff
@@ -103,6 +105,7 @@ func UploadArtifact(
 	chunkSize int64,
 	path string,
 	adpt *adapter.Adapter,
+	silent bool,
```

Comment:
- [ ] I think adding the silent parameter here bloats the parameters unnecessarily. Instead we should use either a specific log level (info or debug), or pass in a copy of the current logger (via NewStdLog()) which then enables/disables specific output.

Reply:
The silent here is primarily to shut-up the progress reported - which is different from logging.

Reply:
In my mind, I seem them being the same, just the target output device is different. You have a log file writer, which includes all the program debug information and then as a subset of that logger, you have just the console output (enabled/disabled if it's something you want to keep a **file** log of). 

In any case, if the silent parameter here is just to a flag to suppress output of this function, I think it can stay.